react-app/src/components/SelectMenu/SelectMenu.tsx

```diff
@@ -72,7 +72,7 @@ export const SelectMenu: FC = () => {
     <div className="bg-gray-50">
       <div className="left-0 max-w-screen-xl px-4 py-12 mx-auto sm:px-6 lg:py-16 lg:px-8 lg:flex lg:items-center lg:justify-between">
         <h2 className="text-2xl font-extrabold leading-9 tracking-tight text-gray-900 sm:text-3xl sm:leading-10">
-          Download cleaned risk free EIOPA rates here:
+          Download risk free EIOPA rates <br/>between 2016 and the most recent month
```

Comment:
Yes, I agree - but today is too ambitious promise because we never offer today's rates, more like 2 weeks after the last month end we can give last month's rates?

But I thought of "last month", "current month" etc. None seems to describe the reality... Alternatively we can perhaps add a * and then explain later what it means by today?

Reply:
That's true - we could, maybe underline the word and then on hover display a tooltip with the caveat?

Reply:
I like this idea!

Reply:
@simicd how did you add a hover-over tooltip? I didn't find one directly from tailwindcss, and the other option I found was write css??

Reply:
I made a proposal (see new commit) - I made it with the Notification component I created the other day and a simple onMouseOver/onMouseOut event handler (I used auto-formatting so you will see that many lines changed but in fact it's only lines 80-94 (event handlers) and 133-143 (Notification component with the text) and 68 (state)

![hover-notification](https://user-images.githubusercontent.com/10134699/89191492-4ad3c700-d5a3-11ea-853a-715e95bf29f0.gif)

Reply:
Cool I like it! I will change once a manth to once a month, otherwise its awesome :D

Reply:
@simicd, In addition, I removed the underline, it takes too much attention I feel...is that ok?

Reply:
Coo, go for it! :)